plonk/src/transcript/mod.rs

```diff
@@ -141,83 +117,77 @@ pub trait PlonkTranscript<F> {
         <Self as PlonkTranscript<F>>::append_message(self, label, &to_bytes!(comm)?)
     }
 
-    /// Append a challenge to the transcript.
-    fn append_challenge<E>(
+    /// Append a field element to the transcript.
+    fn append_field<E>(
```

Comment:
Rename `append_field` to `append_field_element` or `append_field_el`?

Reply:
sounds good, I will change to `append_field_elem()`